docs/section5/Rule5-7-1.md

```diff
@@ -33,4 +33,4 @@
                 - Repeat WWR calculation for User RMR space: `_WWR_user = _calculate_wwr(_user_space)`
                 - **Rule Assertion:** `_WWR_baseline == min(_WWR_user, _allowable_baseline_wwr)`
```

Comment:
Line 26 where we're determining the space conditioning category, please use the methodology discussed for 5-2. It should be based on the heating and cooling capacities.

Reply:
the 40% limit is incorrect for allowable baseline WWR. It is based on Building area type, and is specified in Table G3.1.1-1. Please see the TCD spreadsheet for additional guidance

Reply:
@supriyagoel I think the line that says:

if _building_area_type in table_G3_1_1_1: _allowable_baseline_wwr = data_lookup(table_G3_1_1_1, _building_area_type) else: _allowable_baseline_wwr = 0.40

is already looking up from that table.

Reply:
@supriyagoel description was updated to reflect latest spreadsheet naming.

Reply:
@supriyagoel @JasonGlazer @mkarpman 

I'm a bit confused what we are doing with zones and spaces in the functions related to envelope. The RMR schema still has space_conditioning_category and status_type on Space, but it appears we are using the zone_conditioning_category calculated from Zone.  Since status_type is still on Space how can we confidently determine the Status Type at the zone level?  Does status_type need to be moved to Zone in the schema?

![image](https://user-images.githubusercontent.com/4923674/124019199-97837600-d99d-11eb-9549-c2aed510ff14.png)

Reply:
@mcneillj we are not using conditioning_type for any RDSs - it does not apply to 90.1. Instead, we determine zone conditioning category via a function by applying 90.1 definition to physical elements included in the RMR. 

I don't believe we are using status_type for any RDSs either. I am not sure what it'll be used for. There is also Building.is_all_new which may or may not be useful for anything.

Reply:
@mkarpman That makes sense.  Just clarifying.  I think I'm understanding 5.baseline.c as it states "for new buildings and additions".  Currently, the schema guidance is to model additions as two buildings (one is_all_new==True and one is_all_new==False).  Wouldn't we would need to calculate WWR based on the existing and the addition?  If so, I'm not sure how we are intending to test whether the building has an addition.  Maybe @supriyagoel @JasonGlazer have an idea of this. 

![image](https://user-images.githubusercontent.com/4923674/124044110-75035400-d9c1-11eb-8406-28cd300eebe2.png)

Reply:
I see - I missed that note. I wonder it it makes sense to have this on building level, building segment level or space level. Now we have it in two places - on building level (is_all_new) and space level (status_type). If 90.1 RCT looks at one and not the other, we need to communicate this to the vendors. But ideally I think it should only be in one place. Maybe bring it up with schema WG.